turbo.json

```
@@ -20,17 +20,21 @@
       "outputs": ["dist/**"],
       "dependsOn": ["^build", "compile"]
     },
+    "build:docker": {
+      "inputs": ["Dockerfile", "./"],
+      "outputs": []
+    },
     "test": {
       "inputs": ["src/**", "**/*.unit.ts", "tests-e2e/**"],
```

```suggestion
      "inputs": ["src/**", "src/**/*.unit.ts", "tests-e2e/**"],
```

this was wrongly declared by me